File: src/main/java/me/travis/wurstplusthree/command/commands/BurrowBlockCommand.java
package me.travis.wurstplusthree.command.commands;

import me.travis.wurstplusthree.WurstplusThree;
import me.travis.wurstplusthree.command.Command;
import me.travis.wurstplusthree.hack.combat.Burrow;
import me.travis.wurstplusthree.util.ClientMessage;
import me.travis.wurstplusthree.util.WhitelistUtil;
import net.minecraft.block.Block;

import java.io.IOException;

/**
 * @author Madmegsox1
 * @since 08/05/2021
 */

public class BurrowBlockCommand extends Command {
    public BurrowBlockCommand(){
        super("BurrowBlock", "bb");
    }
    String bBlock = "";

    @Override
    public void execute(String[] message) {
        Burrow bClass = (Burrow) WurstplusThree.HACKS.getHackByName("Burrow");
        Block b = WhitelistUtil.findBlock(message[0]);

        if(b.equals(null)){
            ClientMessage.sendMessage("Cannot set Block to " + message[0]);
            return;
        }
        bClass.setBlock(b);
        ClientMessage.sendMessage("Set Block to " + message[0]);
        bBlock = message[0];
        try {
            WurstplusThree.CONFIG_MANAGER.saveBurrowBlock();
        }catch (IOException e){
        }
    }

    public String getBBlock(){
        return bBlock;
    }

    public void setBBlock(String b){
        bBlock = b;
    }
}

File: src/main/java/me/travis/wurstplusthree/command/commands/BurrowBlockCommand.kt
package me.travis.wurstplusthree.command.commands

import me.travis.wurstplusthree.WurstplusThree
import me.travis.wurstplusthree.command.Command
import me.travis.wurstplusthree.hack.combat.Burrow
import me.travis.wurstplusthree.manager.ConfigManager
import me.travis.wurstplusthree.util.ClientMessage
import me.travis.wurstplusthree.util.WhitelistUtil

/**
 * @author Madmegsox1
 * @since 08/05/2021
 */


class BurrowBlockCommand : Command("BurrowBlock", "bb") {

    var bBlock: String = "";

    override fun execute(message: Array<String>) {
        val bClass = WurstplusThree.HACKS.getHackByName("Burrow") as Burrow
        val b = WhitelistUtil().findBlock(message[0])
        if(b.equals(null)){
            ClientMessage.sendMessage("Cannot set Block to ${message[0]}")
            return
        }
        bClass.block = b
        ClientMessage.sendMessage("Set Block to ${message[0]}")
        bBlock = message[0]
        ConfigManager().saveBurrowBlock()
    }

}